flexbox-practice.html

```diff
@@ -0,0 +1,83 @@
+    <link rel="stylesheet" href="css-flexbox-practice.css">
+</head>
+<body>
+    <div class="main">
+        <div class="site">
+            <nav class="navigation">
+                <img src="./images/sample-logo.jpg" alt="logo">
+                <ul class="nav-list">
+                    <li class="list-item"><a href="#">About US</a></li>
+                    <li class="list-item"><a href="#">Products</a></li>
+                    <li class="list-item"><a href="#">Contacts</a></li>
+                </ul>           
+            </nav>
+            <div class="section-1">
+                <div class="section-1-left">        
+                    <img src="./images/paul-skorupskas.jpg" class="img" alt="image">
+                </div>
+                <div class="section-1-right">
+                    <h2>Title</h2>
+                    <p>Some Text Down Here</p>
+                    <button>Button</button>
+                    <button class="spacing-left">Button</button>
+                </div>
+            </div>
+            <div class="section-2">
+                <div class="box">
+                    <h2>Title</h2>
+                    <p>Some Text</p>
+                </div>
+                <div class="box">
+                    <h2>Title</h2>
+                    <p>Some Text</p>
+                    <button class="btn-2">Button</button>
+                </div>
+                <div class="box">
+                    <h2>Title</h2>
+                    <button>Button</button>
+                    <p>Some Text</p>
+                </div>    
+            </div>
+            <div class="section-3">
+                <h2>Some Heading</h2>
+                <img src="./images/paul-skorupskas.jpg" alt="small image">
+                <p>Text Here Text</p>
+                <p>Here Text Here Text</p>
+                <p>Here Text Here</p>
+            </div>
+            <footer>
+                <div>
+                    <img src="./images/sample-logo.jpg" alt="logo">
+                </div>
+                <div>
+                    <ul class="footer-list">
+                        <li class="footer-links"><a href="#">Link 1</a></li>
+                        <li class="footer-links"><a href="#">Link 2</a></li>
+                        <li class="footer-links"><a href="#">Link 3</a></li>
+                        <li class="footer-links"><a href="#">Link 4</a></li>
+                    </ul>
+                </div>
+                <div>
+                    <ul class="footer-list">
+                        <li class="footer-links"><a href="#">Link 1</a></li>
+                        <li class="footer-links"><a href="#">Link 2</a></li>
+                        <li class="footer-links"><a href="#">Link 3</a></li>
+                        <li class="footer-links"><a href="#">Link 4</a></li> 
+                    </ul> 
+                </div>
+                <div>
+                    <button class="spacing-bottom btn-3">Email</button>
```

Comment:
This should not be a `button` but an `input` within a `<form>` html element:
https://www.w3schools.com/html/html_form_input_types.asp

Add a `placeholder` property to `<input>` with "email" as its value.

Reply:
I put in a form. To get both the space between the form and the space between the buttons I had to use a negative margin. I am sure there is a more elegant way to do this, please give hints or article to this.